integration-cli/docker_cli_config_inspect_test.go

```diff
@@ -10,29 +10,7 @@ import (
 	"github.com/go-check/check"
 )
 
-func (s *DockerSwarmSuite) TestConfigInspect(c *check.C) {
-	d := s.AddDaemon(c, true, true)
-
-	testName := "test_config"
-	id := d.CreateConfig(c, swarm.ConfigSpec{
-		Annotations: swarm.Annotations{
-			Name: testName,
-		},
-		Data: []byte("TESTINGDATA"),
-	})
-	c.Assert(id, checker.Not(checker.Equals), "", check.Commentf("configs: %s", id))
-
-	config := d.GetConfig(c, id)
-	c.Assert(config.Spec.Name, checker.Equals, testName)
-
-	out, err := d.Cmd("config", "inspect", testName)
-	c.Assert(err, checker.IsNil, check.Commentf(out))
-
-	var configs []swarm.Config
-	c.Assert(json.Unmarshal([]byte(out), &configs), checker.IsNil)
-	c.Assert(configs, checker.HasLen, 1)
-}
-
+// TODO: TestConfigInspectMultiple should either be moved to docker/cli or dropped.
```

Comment:
We have pretty good test coverage here: https://github.com/docker/cli/blob/master/cli/command/config/inspect_test.go

I think you can probably remove it now.

Reply:
Thanks @dnephin. Removed.